package.json

```diff
@@ -93,6 +93,7 @@
     "strip-ansi": "^5.2.0"
   },
   "dependencies": {
+    "@serverless/enterprise-plugin": "latest",
```

Comment:
This setup asks for trouble. At least any SFE version with which we will drop support for some non latest SFO may break installations/update of those SFO's versions.

I think we should go regular semver way here, what's the reasoning to go otherwise?

Reply:
hmm. should we bump the plugin to 1.x.x? bc semver is _weird_ for versions < 1

Reply:
>  bc semver is weird for versions < 1

In community coined convention was that then any _minor_  change is breaking but _patch_ updates should not.

Still semver docs state that any (even _patch_) updates can be breaking in v0.x release chain.

I would adapt to what community is used to, and in that case we can pin with `^` to given minor release chain (`^0.10.1` is equal to `>=0.10.1 && <0.11`)

Or we can release v1

Reply:
Yeah, imo if we want to use semver we should be above v1. the semantics of 0.x release makes sense.. in that they're meant for pre-release software effectively, we aren't at that stage any more